.github/workflows/cypress.yml

```diff
@@ -0,0 +1,37 @@
+name: "CYPRESS e2e"
```

Comment:
```suggestion
name: "ZITADEL e2e Test"
```

Reply:
thanks for the notice, I missed on that one